templates/databricks-multi-workspace-oem.template.yaml

```diff
@@ -810,7 +849,15 @@ Resources:
             Action: sts:AssumeRole
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole 
-        - arn:aws:iam::aws:policy/IAMFullAccess 
+        - arn:aws:iam::aws:policy/IAMFullAccess
+      Policies:
+        - PolicyName: kmsUpdateRole
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: 'kms:*'
+                Resource: '*'   
```

Comment:
We need to restrict it to specific resources. In addition, we need to restrict the actions. Do you really need to perform all the actions on the kms resourceS?

Reply:
I think I can limit the actions to:

"Action": [
"kms:Get*",
"kms:PutKeyPolicy"
]

let me test it.

Reply:
Yes please. We need to restrict it. This is against AWS security best practices.